src/commons/interceptors.js

```diff
@@ -44,6 +46,10 @@ export function requestInterceptor(request) {
  * @returns {object}
  */
 export async function responseInterceptor(error) {
+  if (!window.navigator.onLine) {
+    return Promise.reject({ message: errors.NETWORK_ERROR });
```

Comment:
no need for this, 
commented here. https://github.com/leapfrogtechnology/reactjs-starter/pull/27#discussion_r420095047

Reply:
removed this logic